source/command_bus.coffee

```diff
@@ -7,19 +7,22 @@ class Space.messaging.CommandBus extends Space.Object
   }
 
   _handlers: null
+  _onSendHooks: null
 
   constructor: ->
     super
     @_handlers = {}
+    @_onSendHooks = []
 
-  send: (command, callback) ->
+  send: (command, callback, options={}) ->
     if @meteor.isServer
       # ON THE SERVER
       handler = @_handlers[command.typeName()]
-      if not handler?
+      hook(command) for hook in @_onSendHooks when !options.ignoreHooks
```

Comment:
Is this exclusive to testing? If so maybe we can hook up a configuration option. Right now it's part of the public api for sending commands, and ignoring hooks seems like something the sender should have no control over. I'd call this `bypassHooks` too btw